conformance/runner_test.go

```diff
@@ -0,0 +1,221 @@
+package conformance
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/chain/vm"
+	"github.com/filecoin-project/lotus/lib/blockstore"
+
+	"github.com/filecoin-project/statediff"
+	"github.com/filecoin-project/test-vectors/schema"
+
+	"github.com/ipld/go-car"
+	"github.com/logrusorgru/aurora"
```

Comment:
ideally we don't have two different packages imported to print colors. We've been using: https://github.com/fatih/color elsewhere

Reply:
Cool, will switch before merging.

Reply:
Done in https://github.com/filecoin-project/lotus/pull/3081/commits/4709e231b8247f2788122ead8d7f679deadbb51f.